code/modules/client/preference_setup/loadout/loadout_general.dm

```diff
@@ -39,6 +39,28 @@
 	display_name = "spaceball booster pack"
 	path = /obj/item/pack/spaceball
 
+/datum/gear/gamehelm
+	display_name = "handheld video game system"
+	description = "A selection of various GameHelm consoles."
+	cost = 1
+	path = /obj/item/gamehelm
+
+/datum/gear/gamehelm/New()
+	..()
+	var/list/gamehelm = list()
+	gamehelm["red gamehelm"] = /obj/item/gamehelm/red
+	gamehelm["blue gamehelm"] = /obj/item/gamehelm/blue
+	gamehelm["green gamehelm"] = /obj/item/gamehelm/green
+	gamehelm["yellow gamehelm"] = /obj/item/gamehelm/yellow
+	gamehelm["pink gamehelm"] = /obj/item/gamehelm/pink
+	gamehelm["black gamehelm"] = /obj/item/gamehelm/black
+	gamehelm["weathered gamehelm"] = /obj/item/gamehelm/weathered
+	gamehelm["brown gamehelm"] = /obj/item/gamehelm/brown
+	gamehelm["turquoise gamehelm"] = /obj/item/gamehelm/turquoise
+	gamehelm["white gamehelm"] = /obj/item/gamehelm
+	gamehelm["purple gamehelm"] = /obj/item/gamehelm/purple
```

Comment:
wouldn't a greyscale recolourable one work better?

Reply:
Preference of the one who sprited it. I'm PRing this for them as they don't know how to write the code for this. I imagine it's similar to how a GameBoy only comes in certain colours.